app/helpers/users.js

```diff
@@ -13,3 +15,19 @@ exports.encryptPayload = async payload => {
     throw errors.defaultError('Something went wrong trying to encrypt password');
   }
 };
+
+exports.authenticateUser = email => {
+  try {
+    const {
+      common: {
+        session: { secret }
+      }
+    } = config;
```

Comment:
To make this a little cleaner you could do the following: `const {secret} = require ('../../ config'). common.session;`